src/plug/MainView.vala

```diff
@@ -67,38 +63,16 @@ public class PC.MainBox : Gtk.Box {
         main_grid.attach (infobar, 0, 1, 1, 1);
         main_grid.attach (paned, 0, 2, 1, 1);
 
-        var alert_view = new Granite.Widgets.AlertView (_("No users to edit"), _("Parental Controls can only be applied to user accounts that don't have administrative permissions.\nYou can change a user's account type from \"Administrator\" to \"Standard\" in the User Accounts pane."), "preferences-system-parental-controls");
-        alert_view.get_style_context ().remove_class (Gtk.STYLE_CLASS_VIEW);
-
         var link_button = new Gtk.LinkButton.with_label ("settings://accounts", _("Configure User Accounts"));
```

Comment:
Seems this isn't used anywhere anymore, do we still need it?

Reply:
Oops, good catch